docs/en-US/about_Plaster_CreatingAManifest.help.md

```diff
@@ -267,15 +289,17 @@ This example shows Plaster parameter expansion working in the message element:
 ```
 
 ### Content element: Modify
-The modify element allows you to replace file contents, allowing you to copy files using the `file` element, then substituting content to meet your needs.
+The modify element allows you to replace file contents, allowing you to copy files using the `file` element, then substituting content to meet your needs. Multiple original, substitute and condition attributes can be used in a single modify element to modify a file to your requirements.
 
 Available attributes for this content element are:
 - `replace`   - Specify a replacement operation of the file content.
-    - `original`   - The original text, or regular expression match to replace.
+    - `original`   - The original text, or regular expression match to replace. If just searching for text, regular expression syntax must be used by escaping backslashes and other special characters.
+        - `expand` - Whether to expand variables within the original for match.
```

Comment:
We should mention that because XML element content has to encode certain characters you might use in a regex  `<`, `>` that it might be convenient to use a CDATA section to encode the regex pattern.